tests/purs/failing/1175.out

```diff
@@ -15,6 +15,7 @@ while checking that type [33mInt[0m
   is at least as general as type [33mString[0m
 while checking that expression [33m1[0m
   has type [33mString[0m
+while matching label [33mfoo[0m
```

Comment:
This is a bad hint; there are no record labels involved in the code, so understanding this hint requires knowing that the compiler desugars instances to records internally, which is a detail we don't want users to have to know (as it might change).